src/SFA.DAS.AdminService.Web/Infrastructure/IRoatpApplicationApiClient.cs

```diff
@@ -42,8 +46,24 @@ public interface IRoatpApplicationApiClient
 
        
          Task<List<GatewayPageAnswerSummary>> GetGatewayPageAnswers(Guid applicationId);
+
+         //MFCMFC THIS NEEDS TO GO ONCE ALL TIDY UP IS DONE
```

Comment:
Can this be resolved?

Reply:
I'll be resolving this as last step to getting everything clean, probably Monday